README.rst

```diff
@@ -99,6 +99,22 @@ you can do:
 
 .. installation-end-content
 
+Contributing
+============
+
+After cloning this repo, run the following commands to install needed dependencies
+in a proper conda environment:
+
+```
+pip install -r requirements.txt -r dev-requirements.txt
```

Comment:
Isn't that (partially) redundant to what was mentioned above in the section "Install as a Sphinx-Gallery developer"?

Reply:
Just noticed this one and agree -- will push a commit to fix